tests/Callbacks/RequestMetaDataTest.php

```diff
@@ -49,8 +49,28 @@ public function testCanMetaData()
         ]], $report->getMetaData());
     }
 
+    public function testCanConsoleMetaData()
+    {
+        $_SERVER['argv'] = ['some', 'test', 'command', '--opt'];
+
+        $report = Report::fromPHPThrowable($this->config, new Exception())->setMetaData(['bar' => 'baz']);
+
+        $callback = new RequestMetaData($this->resolver);
+
+        $callback($report);
+
+        $this->assertSame(['bar' => 'baz', 'console' => [
+            'Input' => 'some test command --opt',
+            'Command' => 'some',
+            'Arguments' => ['test', 'command'],
+            'Options' => ['--opt'],
+        ]], $report->getMetaData());
```

Comment:
Probably worth also doing an assertion on the report context too.

Reply:
@GrahamCampbell Should there be a test which is different from [this one](https://github.com/bugsnag/bugsnag-php/pull/398/files#diff-f9d661e66db5a9fa0ced0d87ae7ed85fR52)?

Reply:
Oh, ha. No, that looks great. ;)